wireless_modules/config.example.py

```diff
@@ -0,0 +1,9 @@
+from machine import Pin
+
+ESSID = "<WIFI NETWORK NAME HERE>"
+PASSWORD = "<WIFI PASSWORD HERE>"
+
+MQTT_BROKER = "<Broker domain address or IP address>"
+
+builtin_LED = Pin(2, Pin.OUT)
```

Comment:
Could initialising the pin be moved out of the config? So it's just the pin number in here

Reply:
Yeah I can do that. Is this just so that config only has variables and no actual code?

Reply:
Yep :+1: